test/lexers/javascript-lexer.test.js

```diff
@@ -94,13 +101,23 @@ describe('JavascriptLexer', () => {
     done()
   })
 
-  it('supports the acorn-object-rest-spread plugin', (done) => {
-    const Lexer = new JavascriptLexer({ acorn: { plugins: { objectRestSpread: true } } })
+  it('supports the spread operator in objects plugin', (done) => {
```

Comment:
This is the plugin we no longer need 'cause it's supported as long as you set ecmaVersion to 9 or 10.